joomla/joomla-cms
# [4.0] Convert icon- in toolbar to fa- ( part 1 )

Pull Request for Issue #27333 

### Summary of Changes
change icon- classes to be consistent with using fontawesome 5 classes. ( part 1 )

### Testing Instructions
apply pr
run npm ci
edit an article. you will see all the icons in the toolbar.
verify that icons have changed properly and none are missing.


### Expected result
Icons are same as before the pr.

### Documentation Changes Required

none
